bokehjs/src/coffee/models/widgets/slidertemplate.eco

```diff
@@ -1,5 +1 @@
-<label for="<%= @id %>"> <%= @title %>: </label>
-<input type="text" id="<%= @id %>" readonly style="border:0; color:#f6931f; font-weight:bold;">
-<div class="bk-slider-<%=@orientation%>">
-  <div class="slider " id="<%=@id%>">
-</div>
+<input type="text" class="js-range-slider" value="" />
```

Comment:
If this remains untemplated one-liner, remove this and change `SliderView.tagName` to `input` and use `@$el` directly with `ion-rangeslider`.